data/json/professions.json

```diff
@@ -136,8 +136,43 @@
       { "item": "cookbook_human", "bonus": [ [ "CANNIBAL" ] ] },
       { "item": "teleumbrella", "bonus": [ [ "ALBINO" ] ] }
     ]
-  },
-  {
+  },{
+    "type": "profession",
+    "ident": "vagabond",
+    "name": "Vagabond",
+    "description": "Circumstances left you wandering, with no home, no family, no friends.  But the world you knew is gone, and maybe your experiences relying on yourself to survive could be useful in this new one.",
+    "points": 3,
+    "skills": [
+      { "level": 2, "name": "unarmed" },
+      { "level": 2, "name": "survival" },
+      { "level": 2, "name": "tailor" },
+      { "level": 2, "name": "fabrication" },
+      { "level": 2, "name": "cooking" }
+    ],
+    "items": {
+      "both": [
+        "jeans",
+        "tshirt",
+        "gloves_light",
+        "hat_ball",
+        "duffelbag",
+        "long_underpants",
+        "boots",
+        "socks_wool",
+        "socks",
+        "long_undertop",
```

Comment:
Long underwear bottoms are pretty common, but tops are a bit more specialized. I don't think they're as easy to find at thrift stores. Probably best left out.